### PR TITLE
DEVELOPER-1205 Removed the hard coding of the country variable and set it to an environment variable

### PR DESCRIPTION
**Pull request by Dantheman720**

### CODE REVIEW

File: lib/aweplug/books/google_books.rb [hunk 1]
```diff
@@ -51,12 +51,12 @@ def initialize site, push_to_searchisko
       end
 
       def get data
+        env_param = {
+            :q => "isbn:#{data['isbn']}"}
```
Comment (LightGuard) [inline lib/aweplug/books/google_books.rb]: Just a style thing, but either put this hash on one line, or the closing `}` on a different line to make it easier to see where it ends.

**Comment by paulrobinson** [comment]
The CI environment should have the COUNTRY_CODE in the environment. Can you fix up this PR (including a rebase), then raise a PR on JBoss Developer that switches the aweplug version to use this branch? Remember to include the update to Gemfile and Gemfile.lock.

**Comment by LightGuard** [comment]
We need to retest this?

**Comment by Dantheman720** [comment]
Yeah, I am retesting now.  I merged in the code with upstream.

**Comment by Dantheman720** [comment]
@LightGuard This is merged with upstream as of DEVELOPER-1232 changes and is ready for review

**Pull request merged**